structuredefinitions/England-ServiceRequest.xml

```diff
@@ -135,9 +135,6 @@
       <path value="ServiceRequest.basedOn.identifier.value" />
       <min value="1" />
     </element>
-    <element id="ServiceRequest.intent">
```

Comment:
The cardinality of ServiceRequest.intent element is 1..1. Any reason to remove the element?